src/components/ui/Chat/ChatBubble/ChatBubbleContainer.tsx

```diff
@@ -0,0 +1,60 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { FC, HTMLAttributes, ReactNode } from 'react';
+
+import { BaseProps } from '../../Base';
+import { StyledChatBubbleContainer, StyledChatBubbleInfo } from './Styled';
+import PopOver from '../../PopOver';
+import { Dots } from '../../icons';
+import { formatTime } from '../../Utilities';
+
+export type Message = {
+  /* The displayed text of the message sent. */
+  content: string;
+  /* The unique identifier of the message. */
+  messageId: string;
```

Comment:
Why is this prop necessary? It looks like there's a few props that aren't being used in some of these components - I would be careful about tying the components too closely to an SDK data structure, especially in required props

Reply:
Yep, this was supposed to be removed.